node/resolvers/Queries/Organizations.ts

```diff
@@ -105,6 +105,23 @@ const Organizations = {
     }
   },
 
+  getOrganizationsByEmail: async (
+    _: void,
+    { email }: { email: string },
```

Comment:
Rather than allowing any email to be provided as an argument, what if the query had no arguments and the query automatically extracted the email from the current user's session? Is there any situation where one user would need to query another user's organizations?

Reply:
If it's a valid scenario, we need to add a permissions check to see if the current user has permission to view another user's organization data